templates/watcher/config/00-default.conf

```diff
@@ -0,0 +1,43 @@
+[DEFAULT]
+state_path = /var/lib/watcher
+transport_url = {{ .TransportURL }}
+control_exchange = watcher
+debug = True
+
+[database]
+connection = {{ .DatabaseConnection }}
+
+[oslo_policy]
+policy_file = /etc/watcher/policy.yaml.sample
+
+[oslo_messaging_notifications]
+driver = messagingv2
+
+[keystone_authtoken]
+memcached_servers = {{ .MemcachedServers  }}
+# TODO jgilaber implement handling this option when we add tls support
+# cafile = /var/lib/ca-bundle.pem
+project_domain_name = Default
+project_name = service
+user_domain_name = Default
+password = {{ .ServicePassword  }}
+username = {{ .ServiceUser  }}
+auth_url = {{ .KeystoneAuthURL }}
+interface = internal
+auth_type = password
+
+[watcher_clients_auth]
+memcached_servers = {{ .MemcachedServers  }}
+# TODO jgilaber implement handling this option when we add tls support
+# cafile = /var/lib/ca-bundle.pem
+project_domain_name = Default
+project_name = service
+user_domain_name = Default
+password = {{ .ServicePassword  }}
+username = {{ .ServiceUser  }}
+auth_url = {{ .KeystoneAuthURL }}
+interface = internal
+auth_type = password
+
```

Comment:
I would also explicitely set section:
```
[watcher_datasources]
datasources = ceilometer
```

Default is:

datasources = gnocchi,ceilometer,monasca,grafana

But we don't support gnocchi, monasca and grafana in o-k-o so I would explicitely disable them. At some point we may just limit it to prometheus.

Reply:
done, good catch!